src/Bridge/Graphql/Executor.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
Why do we need this class? Can't we call `GraphQL::executeQuery` directly?

Reply:
It's mainly for testability. To be able to mock it.

Reply:
You should be able to mock GraphQL directly then?

Reply:
Indeed, the `GraphQL` class is not final, so you can mock it.

Reply:
Even a static method?

Reply:
Indeed.